modules/local/orthofinder_mcl.nf

```diff
@@ -2,45 +2,33 @@ process ORTHOFINDER_MCL {
     tag "MCL clustering"
     // label 'process_highthread'
     label 'process_medium'
-    container "${ workflow.containerEngine == 'docker' ? 'austinhpatton123/orthofinder-2.5.4_r-4.2.2' : 
+    container "${ workflow.containerEngine == 'docker' ? 'austinhpatton123/orthofinder-2.5.4_r-4.2.2' :
         '' }"
 
     input:
     each mcl_inflation
-    val ch_similarities
-    val mcl_test
+    path(blast)
+    path(fasta)
+    path(db)
+    path(sppIDs)
+    path(seqIDs)
+    val output_directory
 
     output:
-    path "MCL-*-fpath.txt" , emit: og_fpath
+    path("*/Results_Inflation*"), emit: inflation_dir
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
     def args = task.ext.args ?: ''
     def inflation_param = mcl_inflation ? "${mcl_inflation}" : '1.5'
-    def testing_mcl = mcl_test.equals('true') ? "${mcl_test}" : "false"
-    
-    """
-    # add a sleep time equal to the inflation parameter in case things get overloaded when running locally?
-    sleep \$(( \$RANDOM % 10 + 1 ))
 
-    # The following depends on whether we're testing mcl or not. 
-    if [ "$testing_mcl" == "true" ]; then
-        dataDir="${params.outdir}/orthofinder/mcl_test_dataset"
-        cp ${params.outdir}/diamond/TestBlast* .
-        for f in \$(ls TestBlast*)
-        do
-            mv \$f \$(echo \$f | sed "s/TestBlast/Blast/g")
-        done
-    else
-        dataDir="${params.outdir}/orthofinder/complete_dataset"
-        cp ${params.outdir}/diamond/Blast* .
-    fi
-    
-    cp \$dataDir/*dmnd .
-    cp \$dataDir/*fa .
-    cp \$dataDir/*IDs.txt .
+    """
+    for f in \$(ls TestBlast*)
```

Comment:
Lol, yeah, I guess you don't need the if statement at all... So much cleaner....